src/components/generic/MasterContainer.tsx

```diff
@@ -25,6 +25,7 @@ const styles = StyleSheet.create({
     paddingHorizontal: 20,
     marginTop: 10,
   },
+  scrollViewFill: { flexGrow: 1 },
```

Comment:
Waarvoor is dit?

Reply:
had toen te maken met de scrollview die raar deed, maar dat is a opgelost. dit was ook een manier om de scrollview op te lossen. Ik haal het bij deze pr weg.